security/src/main/java/org/kaazing/gateway/security/auth/BasicLoginModule.java

```diff
@@ -61,16 +61,25 @@ public void initialize(Subject subject, CallbackHandler callbackHandler,
     protected boolean doLogin() throws LoginException {
 
         if (!authenticationSchemeIsBasic()) {
-            return true;
+            throw new LoginException("Only able to handle Basic authentication scheme");
```

Comment:
This question to a bunch of other login modules below.  I will only comment here, but if we decide a change is needed we will need to change several other spots